packages/@sanity/vision/src/css/jsonDump.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .string {
```

Comment:
Just to be extra nitpicky, maybe also rename this to `primitive` since it also gets applied to `null`, `number` and `boolean` values?

Reply:
```suggestion
.primitive {
```

Like so?